cmd/disk-cache/req.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"github.com/anujga/dstk/pkg/core"
+)
+
+type RequestType byte
+
+const (
+	Get RequestType = iota
+	Put
+	Remove
+)
+
+type DcRequest struct {
+	K core.KeyT
```

Comment:
why do we need these. can't we just keep the grpc->protobuf object around and implement the `Msg` trait